lib/active_admin/base_controller.rb

```diff
@@ -59,6 +59,10 @@ def active_admin_namespace
     end
     helper_method :active_admin_namespace
 
+    def status_code
+      ActionDispatch::ExceptionWrapper.new(env, env["active_admin.original_error"]).status_code
```

Comment:
I will take it out. The reason why I placed it in the base controller is so that I don't have any logic in the view. I was unable to find any other place to put it.